documentation/source-build.html

```diff
@@ -1,253 +1,240 @@
+                    <pre>source .profile</pre>
+                    The JAVA_HOME variable should now be configured correctly.
+            </ol>
 
-    To execute a build pahse, type <code>mvn "phase"</code> where "phase" is
-    one of the keywords listed below:
+            <h4>Microsoft Windows</h4>
+            <ol>
+                <li>Navigate to your desktop and right click on "My Computer"; choose properties.
+                <li>Select the "Advanced" tab and click on the "Environment Variables" button.
+                <li>Click the "New..." button in the System variables section.
+                    Enter the variable name "JAVA_HOME" and set the variable
+                    value to the full path of your Java installation. For example,
+                    "c:\jdk11". Be sure to not add an extra slash to the end
+                    of the directory name.
+                <li>Click "OK" in all the menus to accept the changes.
+                <li>Close any open command prompt windows. The next time you
+                    open a command prompt, the "JAVA_HOME" variable will be set
+                    correctly.
+            </ol>
 
-    <ul>
-        <li><a href="#clean">clean</a>
-        <li><a href="#compile">compile</a>
-        <li><a href="#test">test</a>
-        <li><a href="#package">package</a>
-    </ul>
+        </section>
 
-    <br><br>
+        <section id="maven">
 
-    Note that you can combine several phases, for example: <code>mvn clean package</code>.
-    When a phase is given, Maven will execute every phase in the sequence up to and
-    including the one defined. For example, when the 'package' phase is executed, the 
-    'compile' and 'test' phases are also executed (but the 'clean' phase is not, unless
-    specified explicitly). Please refer to the Maven documentation for more details.
+            <h3>Install the Maven Build Tool</h3>
 
-<p>
+            <p>
+                The Openfire build process uses Maven, so that tool must be installed and configured on your computer.
+                First download Maven from: <a href="https://maven.apache.org/">https://maven.apache.org</a>. Next,
+                follow the <a href="https://maven.apache.org/users/index.html">usage instructions</a>.
+            </p>
 
+            <h4>Test building with Maven</h4>
 
-<!--CLEAN-->
-<p><b><a name="clean">clean</a></b>
-<ul>
-<i>Syntax:</i><p>
+            <p>
+                Navigate into the root directory of this distribution named via the command-line. In that directory,
+                invoke the build tool to compile the Openfire source code:
+            </p>
+            <pre>mvn compile</pre>
+            <p>
+                If the build tool is invoked correctly and Openfire compiles, you've correctly
+                configured Maven and your copy of the Openfire source distribution.
+            </p>
 
-<span class="code"> mvn clean <br></span>
+        </section>
```

Comment:
Should we ditch the maven section now that we've got the wrappers properly in place?

Reply:
(which reminds be that buidWithDocker.sh is almost certainly not using the wrapper - must create a ticket for this)

Reply:
I wouldn't be surprised if most use the non-wrapper anyway (I know I do). Maybe we should _add_ the wrapper to this text?